lib/internal/crypto/random.js

```diff
@@ -119,6 +120,51 @@ function randomFill(buf, offset, size, cb) {
   _randomBytes(buf, offset, size, wrap);
 }
 
+// Largest integer we can read from a buffer.
+// e.g.: Buffer.from("ff".repeat(6), "hex").readUIntBE(0, 6);
+const RAND_MAX = 281474976710655;
+
+function randomInt(min, max, cb) {
```

Comment:
I am not a big fan of `min` being a mandatory argument. In fact, it might make sense not to specify `min` at all, and to always assume it to be zero. It's trivial to account for that as the caller.

Reply:
I pushed new version where min is optional with default value of `0`.